File: user/src/main/java/com/kjj/user/entity/UserMyPage.java
package com.kjj.user.entity;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Entity
@Getter
@AllArgsConstructor
@NoArgsConstructor
public class UserMyPage {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private int point;

    public void updatePoint(int value) {
        point += value;
    }

    public static UserMyPage createNewUserMyPage() {
        return new UserMyPage(
                null,
                0
        );
    }
}

File: user/src/main/kotlin/com/kjj/user/entity/UserMypage.kt
package com.kjj.user.entity

import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.GenerationType
import jakarta.persistence.Id

@Entity
class UserMypage(
   @Id
   @GeneratedValue(strategy = GenerationType.IDENTITY)
   val id: Long?,
   var point: Int
) {
    companion object {
        fun createNewUserMyPage(): UserMypage {
            return UserMypage(
                null,
                0
            )
        }
    }

    fun updatePoint(value: Int) {
        point += value
    }
}